api/src/main/java/org/bahmni/module/fhircdss/api/service/impl/ConditionsRequestBuilder.java

```diff
@@ -111,10 +112,11 @@ private void addExistingActiveConditions(Bundle conditionsBundle, String patient
         referenceParam.setValue(patientUuid);
         referenceAndListParam.addValue(new ReferenceOrListParam().add(referenceParam));
 
-        IBundleProvider iBundleProvider = fhirConditionService.searchConditions(referenceAndListParam, null,
-                null, null, null,
+        ConditionSearchParams conditionSearchParams = new ConditionSearchParams(referenceAndListParam, null,
                 null, null, null, null,
-                null);
+                null, null, null, null);
```

Comment:
@mohan can we use named parameter here if possible

Reply:
we can name parameters once we pass on some values,